aries-site/src/data/structures/learn.js

```diff
@@ -54,6 +55,21 @@ export const learn = [
     relatedContent: [],
     tags: [],
   },
+  {
+    name: 'How to add additional controls to a toolbar',
+    description:
+      'This how-to guide shows you how compose a Toolbar when controls beyond search and filter are needed.',
+    type: 'How-to guides',
+    preview: {
+      component: () => <DataHowToAddAdditionalControls />,
+      background: 'background-front',
+    },
+    seoDescription:
+      'This how-to guide shows you how compose a Toolbar when controls beyond search and filter are needed.',
+    sections: [],
+    relatedContent: [],
```

Comment:
do we not add related content in learn pages? maybe even just Data for now and Filtering

Reply:
Good call, will add.

Reply:
Done.